### PR TITLE
Cookie Fixes into Development Branch

### PR DESCRIPTION
**Pull request by cjstehno**
@dwclark we should be merging this into `development` rather than `master` at this point. I have other changes there and will merge my branch into `development` and then cut a release. Unless you had other reasons for wanting to go directly into `master`.

### CODE REVIEW

File: http-builder-ng-core/src/main/java/groovyx/net/http/FromServer.java [hunk 1]
```diff
@@ -404,7 +422,8 @@ public Long parse() {
             tmp.put("Refresh", CombinedMap::new);
             tmp.put("Retry-After", HttpDate::new);
             tmp.put("Server", ValueOnly::new);
-            tmp.put("Set-Cookie", MapPairs::new);
+            tmp.put("Set-Cookie", HttpCookies::new);
+            tmp.put("Set-Cookie2", HttpCookies::new);
```
Comment (cjstehno) [inline http-builder-ng-core/src/main/java/groovyx/net/http/FromServer.java]: Hmm... looks like this is real, though obsolete, nevermind.

**Comment by cjstehno** [comment]
Looks like this cleaned up the code a bit too. Looks good to me.

**Comment by dwclark** [comment]
Merging into development was what I meant to say.

File: http-builder-ng-core/src/main/java/groovyx/net/http/FromServer.java (hunk 1 above)
Comment (dwclark) [inline http-builder-ng-core/src/main/java/groovyx/net/http/FromServer.java]: Set-Cookie2 is actually the signal that the server is expecting version 1 of cookies, not the version 0 netscape spec for cookies.

**Pull request merged**